lib/typescript/render/shared.ts

```diff
@@ -1,6 +1,5 @@
-import {isFromContact, RenderedContent, Contact} from 'httpclient';
-import {formatTime} from 'dates';
-import {DefaultMessageRenderingProps} from './components';
+import {isFromContact, RenderedContentUnion} from 'httpclient';
```

Comment:
nitpick: `shared.ts` was never a great name for this file. Looking at it, what do you think about `props.ts` or something else?

Reply:
yeah I think _props.ts_ makes sense :)